frontend/admin/src/js/components/pool/CreatePool.tsx

```diff
@@ -188,185 +171,82 @@ export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
-              rules={{
-                required: intl.formatMessage(errorMessages.required),
-              }}
-            />
-            <MultiSelect
-              id="essentialCriteria"
-              label={intl.formatMessage({
-                defaultMessage: "Essential Criteria",
-                description:
-                  "Label displayed on the pool form essential criteria field.",
-              })}
-              placeholder={intl.formatMessage({
-                defaultMessage: "Select one or more essential...",
-                description:
-                  "Placeholder displayed on the pool form essential criteria field.",
-              })}
-              name="essentialCriteria"
-              options={cmoAssetOptions}
-              rules={{
-                required: intl.formatMessage(errorMessages.required),
-              }}
-            />
-            <MultiSelect
-              id="operationalRequirements"
-              name="operationalRequirements"
-              label={intl.formatMessage({
-                defaultMessage: "Operational Requirements",
-                description:
-                  "Label displayed on the pool form operational requirements field.",
-              })}
-              placeholder={intl.formatMessage({
+              context={intl.formatMessage({
                 defaultMessage:
-                  "Select one or more operational requirements...",
-                description:
-                  "Placeholder displayed on the pool form operational requirements field.",
-              })}
-              options={OperationalRequirementV2.map((value) => ({
-                value,
-                label: intl.formatMessage(getOperationalRequirement(value)),
-              }))}
-              rules={{
-                required: intl.formatMessage(errorMessages.required),
-              }}
-            />
-            <TextArea
-              id="keyTasks_en"
-              name="keyTasks.en"
-              label={intl.formatMessage({
-                defaultMessage: "Key Tasks (English)",
-                description:
-                  "Label displayed on the pool form key tasks (English) field.",
+                  "The 'key' is a string that uniquely identifies a Pool. It's auto-generated based on the pool's group and job title.",
+                description: "Context displayed on the pool form key field.",
               })}
             />
-            <TextArea
-              id="keyTasks_fr"
-              name="keyTasks.fr"
-              label={intl.formatMessage({
-                defaultMessage: "Key Tasks (French)",
+            <Submit
+              text={intl.formatMessage({
+                defaultMessage: "Create new pool",
                 description:
-                  "Label displayed on the pool form key tasks (French) field.",
+                  "Label displayed on submit button for new pool form.",
               })}
             />
```

Comment:
Looks like the mockup uses the `cta` colour option. Could we add that here?

Reply:
ah, I somehow missed the button colour, added